src/windows/cheats.c

```diff
@@ -132,8 +132,8 @@ enum WINDOW_CHEATS_WIDGET_IDX {
 #define YOS				TAB_HEIGHT + YSPA								// Y offset from top (includes tabs height)
 #define BTNW			110												// button width
 #define BTNH			16												// button height
-#define OPTW			220												// Option (checkbox) width (two colums)
-#define OPTH			10												// Option (checkbox) height (two colums)
+#define OPTW			220												// Option (checkbox) width (two columns)
+#define OPTH			10												// Option (checkbox) height (two columns)
```

Comment:
I honestly don't see any trailing whitespaces, I added the "n" in each word by hand on that one.
In my IDE it shows there's nothing after the ")"